scripts/scr_inventory_init.gml

```diff
@@ -2,7 +2,8 @@
 active = false;
 maxItemCount = 10;
 
-selected = 0;
+selectX = 0;
+selectY = 0;
```

Comment:
You could add constants here to define how big a row is and how big a column is. I see you hard coding 2 and 5 in a few places. Having it in one place makes sure that if you ever change it or ever want to use it in another spot, then you don't have to worry about updating all the places. It also makes your intent clearer.